x-pack/platform/plugins/shared/ml/public/application/explorer/explorer_charts/components/explorer_chart_label/entity_filter/entity_filter_styles.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { css } from '@emotion/react';
+
+import { useEuiTheme } from '@elastic/eui';
+
+export const useEntityFilterStyles = () => {
+  const { euiTheme } = useEuiTheme();
+
+  return {
+    filterButton: css({
+      opacity: 0.5,
+      width: euiTheme.size.base,
+      height: euiTheme.size.base,
+      '-webkit-transform': 'translateY(-1px)',
```

Comment:
I'm seeing errors in the console which are being caused by these `-webkit-transform` rules.
![image](https://github.com/user-attachments/assets/2fed2b34-8da5-4b06-ae3c-f2d58c0ef665)

The errors do not appear if these are removed.

Reply:
Removed in https://github.com/elastic/kibana/pull/212793/commits/ac5e22e1e8c3efcbb26b20b90f03f4cf3fd2dd6c